{"uuid": "c5f89541-d9f5-4547-8949-594e044e0a41", "children": ["401e6638-2917-4e69-8f9c-b37bf1119b36", "bc2f1678-fb8e-47f2-88e6-3a9912aef8f1", "cdb3ddff-9c5d-4932-adff-29c56af2310c", "15909e01-7ced-441f-a8d8-1dbebf57e629", "851b46c1-7459-4372-92b6-27b3612172f6", "bf52ffc0-c285-4639-9a54-c46d39b94222"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'wlan0-1'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 408, in get_interface_details\n    wifi_info[i].append(data[i])\n"}, "attachments": [{"name": "ucentral_config: ", "source": "2912bf50-d41b-4003-823c-f7fb687d8b1d-attachment.attach"}, {"name": "Time Took to apply Config: 56.00445103645325", "source": "c84ae3cc-49a1-46f4-a8a1-ac0c4dba4689-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "4ea30a8b-9973-4505-8958-74154a0bfc1a-attachment.attach"}, {"name": "AP LOgs: ", "source": "79339c26-294a-42ce-9e09-d94e4b937671-attachment.attach"}], "start": 1629506099980, "stop": 1629506188562}], "start": 1629506099980, "stop": 1629506188878}